tests/functional/splinter_test.c

```diff
@@ -2612,8 +2614,9 @@ splinter_test(int argc, char *argv[])
       config_argv += 1;
    }
    if (config_argc > 0
-       && strncmp(
-             config_argv[0], "--num-bg-threads", sizeof("--num-bg-threads"))
+       && strncmp(config_argv[0],
```

Comment:
Should be handled by `config.[hc]` parser.

Reply:
@rtjohnso - That seems like a general comment, no? Or are you suggesting that I take on this comment and implement it in this PR?

We've talked several times in PR reviews that this config stuff should be overhauled, majorly. Please log a new issue, if one does not already exist ... but I am not sure that overhaul belongs to this enablement PR.

Reply:
See response above.